HoaryFox/KarambaConnect/CrossSection.cs

```diff
@@ -0,0 +1,195 @@
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using Karamba.CrossSections;
+using Karamba.Materials;
+using STBReader;
+using STBReader.Member;
+using STBReader.Section;
+
+namespace HoaryFox.KarambaConnect
+{
+    public class CrossSection
```

Comment:
Codacy found an issue: [Add a 'protected' constructor or the 'static' keyword to the class declaration.](https://app.codacy.com/gh/hrntsm/HoaryFox/pullRequest?prid=6405207)

Reply:
fix it